recipes/git-python/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "git-python" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/source/g/{{ name }}/{{ name }}-{{ version }}.zip
```

Comment:
Oh, this is a PyPI URL, is it?

Reply:
I had to fall back a couple versions, but it was there! I'm not sure why the package creator stopped uploading the source after version 1.0.3